Pod/Classes/LSPAudioPlayerModel.h

```diff
@@ -0,0 +1,24 @@
+//
+//  LSPAudioPlayerModel.h
+//  Expecta
```

Comment:
What does Expecta mean?

Reply:
Xcode issue. Still belongs to the `loudspeaker` bundle. Will resolve.